2020-10-23 19:12:59,236 INFO    MainThread:143818 [wandb_init.py:_log_setup():292] Logging user logs to wandb/run-20201023_191259-3sixr3p8/logs/debug.log
2020-10-23 19:12:59,236 INFO    MainThread:143818 [wandb_init.py:_log_setup():293] Logging internal logs to wandb/run-20201023_191259-3sixr3p8/logs/debug-internal.log
2020-10-23 19:12:59,236 INFO    MainThread:143818 [wandb_setup.py:_flush():68] enabling code saving by default
2020-10-23 19:12:59,236 INFO    MainThread:143818 [wandb_setup.py:_flush():68] setting env: {}
2020-10-23 19:12:59,236 INFO    MainThread:143818 [wandb_setup.py:_flush():68] setting user settings: {'save_code': True}
2020-10-23 19:12:59,236 INFO    MainThread:143818 [wandb_setup.py:_flush():68] multiprocessing start_methods=fork,spawn,forkserver
2020-10-23 19:13:02,386 INFO    MainThread:143818 [wandb_run.py:_console_start():1265] atexit reg
2020-10-23 19:13:02,387 INFO    MainThread:143818 [wandb_run.py:_redirect():1135] redirect: SettingsConsole.REDIRECT
2020-10-23 19:13:02,388 INFO    MainThread:143818 [wandb_run.py:_redirect():1138] Redirecting console.
2020-10-23 19:13:02,388 INFO    MainThread:143818 [redirect.py:install():196] install start
2020-10-23 19:13:02,389 INFO    MainThread:143818 [redirect.py:install():211] install stop
2020-10-23 19:13:02,389 INFO    MainThread:143818 [redirect.py:install():196] install start
2020-10-23 19:13:02,389 INFO    MainThread:143818 [redirect.py:install():211] install stop
2020-10-23 19:13:02,389 INFO    MainThread:143818 [wandb_run.py:_redirect():1182] Redirects installed.
2020-10-23 19:13:04,784 INFO    Dummy-5   :143818 [wandb_run.py:_atexit_cleanup():1238] got exitcode: 1
2020-10-23 19:13:04,789 INFO    Dummy-5   :143818 [wandb_run.py:_restore():1210] restore
2020-10-23 19:13:04,789 INFO    Dummy-5   :143818 [redirect.py:uninstall():215] uninstall start
2020-10-23 19:13:04,790 INFO    Dummy-5   :143818 [redirect.py:_stop():264] _stop: stdout
2020-10-23 19:13:04,790 INFO    Dummy-5   :143818 [redirect.py:_stop():270] _stop closed: stdout
2020-10-23 19:13:04,790 INFO    stdout    :143818 [redirect.py:_pipe_relay():119] relay done saw last write: stdout
2020-10-23 19:13:04,791 INFO    stdout    :143818 [redirect.py:_pipe_relay():135] relay done done: stdout
2020-10-23 19:13:04,791 INFO    Dummy-5   :143818 [redirect.py:_stop():276] _stop joined: stdout
2020-10-23 19:13:04,791 INFO    Dummy-5   :143818 [redirect.py:_stop():278] _stop rd closed: stdout
2020-10-23 19:13:04,791 INFO    Dummy-5   :143818 [redirect.py:uninstall():219] uninstall done
2020-10-23 19:13:04,791 INFO    Dummy-5   :143818 [redirect.py:uninstall():215] uninstall start
2020-10-23 19:13:04,791 INFO    Dummy-5   :143818 [redirect.py:_stop():264] _stop: stderr
2020-10-23 19:13:04,791 INFO    Dummy-5   :143818 [redirect.py:_stop():270] _stop closed: stderr
2020-10-23 19:13:04,791 INFO    stderr    :143818 [redirect.py:_pipe_relay():119] relay done saw last write: stderr
2020-10-23 19:13:04,791 INFO    stderr    :143818 [redirect.py:_pipe_relay():135] relay done done: stderr
2020-10-23 19:13:04,791 INFO    Dummy-5   :143818 [redirect.py:_stop():276] _stop joined: stderr
2020-10-23 19:13:04,791 INFO    Dummy-5   :143818 [redirect.py:_stop():278] _stop rd closed: stderr
2020-10-23 19:13:04,791 INFO    Dummy-5   :143818 [redirect.py:uninstall():219] uninstall done
2020-10-23 19:13:04,798 INFO    Dummy-5   :143818 [wandb_run.py:_wait_for_finish():1349] got exit ret: uuid: "b69be245cdfa45e1a8718305dc616e89"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 2
      other_count: 1
    }
    pusher_stats {
      uploaded_bytes: 99342
      total_bytes: 99342
    }
  }
}

2020-10-23 19:13:06,803 INFO    Dummy-5   :143818 [wandb_run.py:_wait_for_finish():1349] got exit ret: uuid: "c4c4754970d44ff2b4589dc62dff76d8"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 6
      other_count: 1
    }
    pusher_stats {
      uploaded_bytes: 101379
      total_bytes: 101379
    }
  }
}

2020-10-23 19:13:07,912 INFO    Dummy-5   :143818 [wandb_run.py:_show_files():1543] logging synced files
2020-10-23 19:13:07,913 INFO    Dummy-5   :143818 [internal.py:handle_exit():137] Internal process exited
